frontend/src/components/custom-tools/prompt-card/PromptOutput.jsx

```diff
@@ -244,82 +283,39 @@ function PromptOutput({
-                        <span>
-                          Enabled
-                          <CheckCircleOutlined
-                            style={{
-                              color: "#52c41a",
-                              marginLeft: "5px",
-                            }}
-                          />
-                        </span>
-                      ) : (
-                        <span>
-                          Disabled
-                          <ExclamationCircleFilled
-                            style={{
-                              color: "#BABBBC",
-                              marginLeft: "5px",
-                            }}
-                          />
-                        </span>
-                      )}
-                    </CheckableTag>
-                    <div className="llm-info-container">
-                      <Tooltip title={tooltipContent(profile?.conf)}>
-                        <InfoCircleOutlined className="prompt-card-actions-head" />
-                      </Tooltip>
-                      <Tooltip title="Chunk used">
-                        <DatabaseOutlined
-                          onClick={() => {
-                            setIsIndexOpen(true);
-                            setOpenIndexProfile(promptOutputData?.context);
-                          }}
-                          className="prompt-card-actions-head"
-                        />
-                      </Tooltip>
-                      {ChallengeModal && (
-                        <ChallengeModal
-                          challengeData={promptOutputData?.challengeData || {}}
-                          context={promptOutputData?.context || ""}
-                          tokenUsage={promptOutputData?.tokenUsage || {}}
-                        />
-                      )}
-                      {isNotSingleLlmProfile && (
-                        <Radio
-                          checked={profileId === selectedLlmProfileId}
-                          onChange={() => handleSelectDefaultLLM(profileId)}
-                          disabled={isPublicSource}
-                        >
-                          Default
-                        </Radio>
-                      )}
+                    <div>
+                      <CheckableTag
+                        checked={isChecked}
+                        onChange={(checked) =>
+                          handleTagChange(checked, profileId)
+                        }
+                        disabled={isPublicSource}
+                        className={isChecked ? "checked" : "unchecked"}
+                      >
+                        {isChecked ? (
+                          <span>
+                            Enabled
+                            <CheckCircleOutlined
+                              style={{
+                                color: "#52c41a",
+                                marginLeft: "5px",
+                              }}
```

Comment:
@tahierhussain  Is it possible to move this to css file?

Reply:
@Deepak-Kesavan Thanks for the catch! I have moved it to the CSS file.